src/cmd/markov.py

```diff
@@ -180,3 +198,74 @@ async def _inspectmarkov(self, cmd_line: List[str], execution_ctx: ExecutionCont
+        bc.markov.filters.append(re.compile(cmd_line[1], re.DOTALL))
+        await Command.send_message(execution_ctx, f"Filter '{cmd_line[1]}' was successfully added for Markov model")
+
+    async def _listmarkovfilter(self, cmd_line: List[str], execution_ctx: ExecutionContext):
+        """Print list of regular expression filters for Markov model
+    Example: !listmarkovfilter"""
+        if not await Command.check_args_count(execution_ctx, cmd_line, min=1, max=1):
+            return
+        result = ""
+        for index, regex in enumerate(bc.markov.filters):
+            result += f"{index} -> `{regex.pattern}`\n"
+        await Command.send_message(execution_ctx, result or "No filters for Markov model found!")
+        return result
+
+    async def _delmarkovfilter(self, cmd_line: List[str], execution_ctx: ExecutionContext):
+        """Delete regular expression filter for Markov model by index
+    Example: !delmarkovfilter 0"""
+        if not await Command.check_args_count(execution_ctx, cmd_line, min=2, max=2):
+            return
+        index = await Util.parse_int_for_command(
+            execution_ctx, cmd_line[1],
+            f"Second parameter for '{cmd_line[0]}' should be an index of filter")
+        if index is None:
+            return
+        if 0 <= index < len(bc.markov.filters):
+            bc.markov.filters.pop(index)
+            await Command.send_message(execution_ctx, "Successfully deleted filter!")
+        else:
+            await Command.send_message(execution_ctx, "Invalid index of filter!")
+
+    async def _addmarkovignoredprefix(self, cmd_line: List[str], execution_ctx: ExecutionContext):
+        """Add message prefix that should be ignored by Markov model
+    Example: !addmarkovignoredprefix $"""
+        if not await Command.check_args_count(execution_ctx, cmd_line, min=2):
+            return
+        prefix = ' '.join(cmd_line[1:])
+        index = bc.config.ids["markov_ignored_prefix"]
+        bc.markov.ignored_prefixes[index] = ' '.join(cmd_line[1:])
+        bc.config.ids["markov_ignored_prefix"] += 1
+        await Command.send_message(execution_ctx, f"Added '{prefix}' as ignored prefix for Markov model")
+
+    async def _listmarkovignoredprefix(self, cmd_line: List[str], execution_ctx: ExecutionContext):
+        """List all prefixes that should be ignored by Markov model
+    Example: !listmarkovignoredprefix"""
+        if not await Command.check_args_count(execution_ctx, cmd_line, min=1, max=1):
+            return
+        result = ""
+        for index, prefix in bc.markov.ignored_prefixes.items():
+            result += f"{index} -> `{prefix}`\n"
+        await Command.send_message(execution_ctx, result or "No ignored prefixes for Markov model found!")
+
+    async def _delmarkovignoredprefix(self, cmd_line: List[str], execution_ctx: ExecutionContext):
+        """Delete message prefix that should be ignored by Markov model by its index
+    Example: !delquote 0"""
```

Comment:
Just noticed in on review of your PR. This command has an incorrect example in the docstring. Could you please fix?

Reply:
Fixed